crates/api_models/src/ephemeral_key.rs

```diff
@@ -0,0 +1,8 @@
+use serde;
+
+#[derive(Debug, serde::Serialize, serde::Deserialize, Clone, Eq, PartialEq)]
+pub struct EphemeralKeyCreateResponse {
+    pub created_at: i64,
+    pub expires: i64,
+    pub secret: String,
+}
```

Comment:
Ephemeral key is specific to one customer not I feel it's better to have it in the response json. 

cc: @bernard-eugine

Reply:
You mean, we should only send EphemeralKey as a string in the response? without created_at and expires fields.

Reply:
@hrithikesh026  I meant we should add customerid in the response since it's only specific to that customer.

Reply:
Yeah @hrithikesh026 since it's specific to one customer I think it must be in that block of response itself

Reply:
I have made the above mentioned change.